lib/jsonPointer.js

```diff
@@ -182,6 +182,17 @@ function removeLocalReferenceFromPath(refValue) {
   return refValue.$ref;
 }
 
+/**
+ * Removes local pointer inside a path
+ * Unescape ~1 character
+ * @param {string} refValue - value of the $ref property
+ * @returns {string} - the calculated and unescaped path
+ */
+function pathSolver(refValue) {
+  const path = removeLocalReferenceFromPath(refValue);
+  return path.replace(/~1/g, '/');
```

Comment:
@webholik There can be more escape characters used and we should also resolve them. Is there some standard APIs or library we can use to unescape here instead?

Reply:
@VShingala Only `~` and `/` are escaped in JSON pointers. I have added handling for `~` as well.